appveyor.yml

```diff
@@ -0,0 +1,42 @@
+version: 1.0.{build}
+max_jobs: 2
+image: Visual Studio 2015
+build_script:
+- cmd: >-
+    git submodule init
+
+    git submodule update
+
+    nuget restore ReactWindows\ReactNative.sln
+
+    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
+
+    %DEVENV% /build "Debug|x64" ReactWindows\ReactNative.sln
+
+    npm install -g npm
+
+    npm version
+
+    npm install -g react-native-cli
+
+    npm install
```

Comment:
We can get rid of it, but caching node_modules with AppVeyor brings it down. Happy to ditch it for now.